src/xenialdan/PocketRadio/EventListener.php

```diff
@@ -4,14 +4,13 @@
 
 use pocketmine\event\Listener;
 use pocketmine\event\player\PlayerJoinEvent;
-use pocketmine\plugin\Plugin;
 
 class EventListener implements Listener
 {
     /** @var Loader */
     public $owner;
 
-    public function __construct(Plugin $plugin)
+    public function __construct(Loader $plugin)
```

Comment:
???

Reply:
Change with sense, within this class the variable `$tasks` is used which is not found in the `Loade`r class, there is no variable called `$tasks` in the `Plugin` class.

Anyway this change is not related to the main objective of this PR, I will remove it.